config/plugins/sonic-pbh_yang.py

```diff
@@ -0,0 +1,747 @@
+"""
+This CLI plugin was auto-generated by using 'sonic-cli-gen' utility, BUT
+it was manually modified to meet the PBH HLD requirements.
+
+PBH HLD - https://github.com/Azure/SONiC/pull/773
+CLI Auto-generation tool HLD - https://github.com/Azure/SONiC/pull/78
+"""
+
+import click
+import ipaddress
+import re
+import utilities_common.cli as clicommon
+import utilities_common.general as general
+from ipaddress import IPv4Address, IPv6Address
+
+
+hash_field_types = ['INNER_IP_PROTOCOL',
+                    'INNER_L4_DST_PORT',
+                    'INNER_L4_SRC_PORT',
+                    'INNER_DST_IPV4',
+                    'INNER_SRC_IPV4',
+                    'INNER_DST_IPV6',
+                    'INNER_SRC_IPV6']
+
+packet_action_types = ['SET_ECMP_HASH', 'SET_LAG_HASH']
+
+flow_counter_state = ['DISABLED', 'ENABLED']
+
+pbh_pattern = r"(0x){1}[a-fA-F0-9]+/(0x){1}[a-fA-F0-9]+"
```

Comment:
@vadymhlushko-mlnx we have different patterns: https://github.com/Azure/sonic-buildimage/pull/7461